certified-connectors/DocuSignDemo/script.csx

```diff
@@ -89,7 +89,7 @@ public class Script : ScriptBase
         { "signHereTabs", "Signature" }, 
         { "signerAttachmentTabs", "Signer Attachment" },
         { "ssnTabs", "SSN" },
-        { "tabGroups", "Tab Group"},
```

Comment:
Do we have "Tab Group" somewhere else?

Reply:
Product wants to replace the wording "Tab Group" with "Checkbox Group", thus this change. But we are still using tabGroups, only the UI text changes.